gradlew

```diff
@@ -175,7 +175,7 @@ APP_ARGS=$(save "$@")
 eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
 
 # by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
-if [[ "$(uname)" == "Darwin" ]] && [[ "$HOME" == "$PWD" ]]; then
+if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
```

Comment:
why is the equality check changed from `==` to `=`?

Reply:
This was a side effect of the upgrade to a newer gradle version. When I tried to put it back to the old equality expression it gave me a bizarre whining about that line.
./gradlew: 178: [: Linux: unexpected operator

 So I checked on the pravega main project gradlew and it also was like the upgraded one  - 
if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then